docs/README.md

```diff
@@ -43,7 +43,7 @@
 * [`NodeLocalDNS` feature](usage/node-local-dns.md)
```

Comment:
While you are on this file, there is a link https://www.sap-tv.com/video/40962/gardener-planting-the-seeds-of-success-in-the-cloud to a resource that seems to be no longer existing, can you please take a look?